core/src/main/java/com/netflix/conductor/core/events/DefaultEventQueueManager.java

```diff
@@ -126,7 +126,7 @@ public void doStop() {
                 });
     }
 
-    @Scheduled(fixedDelay = 60_000)
+    @Scheduled(initialDelay = 10_000, fixedDelay = 60_000)
```

Comment:
Adding a delay for initial kafka connection to start once the app is at a more stable state